Earthfile

```diff
@@ -33,4 +33,30 @@ release-meta:
 scripts:
     FROM scratch
     COPY scripts /usr/src/scripts
-    SAVE ARTIFACT /usr/src/scripts scripts
\ No newline at end of file
+    SAVE ARTIFACT /usr/src/scripts scripts
+
+UPLOAD_LOGS:
+    FUNCTION
+    ARG PULL_REQUEST
+    ARG BRANCH
+    ARG COMMIT_HASH
+    LOCALLY
+    LET COMMIT_HASH="${COMMIT_HASH:-$(git rev-parse HEAD)}"
+    FROM +base-log-uploader
+    COPY ./log /usr/var/log
```

Comment:
We should probably make `./log` an arg if we're going to call it from different places

Reply:
I've added an argument but defaulted it to `./log`